http/http.go

```diff
@@ -95,15 +115,17 @@ func (s Server) getIndex(w http.ResponseWriter, r *http.Request) {
 
 	p, err := s.DB.ReadPosts(par)
 	if err != nil {
-		log.Fatalf("read posts: %v", err)
+		s.handleHTMLError(w, r, "read posts", 404, err)
+		return
```

Comment:
👍 

For simplicity, try to do 

```suggestion
		return s.handleHTMLError(w, r, "read posts", 404, err)
```

Unsure if it'll work, but might be worth the simplification.

Reply:
I tried before and it gave me errors due to not enough values assigned or smth like that. 
Will try again.